topbeat/system/sigar.go

```diff
@@ -0,0 +1,19 @@
+package system
```

Comment:
I am not sure if `sigar.go` is the best name for this file especially after you split the sigar functions into multiple dedicated files. As it contains for now only `Round` function, I would go for `common.go` and it can be considered the place for all the common system functions.